src/scripts/main.js

```diff
@@ -354,7 +354,40 @@
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+'use strict';
+
+const table = document.querySelector('.dashboard');
+
+function createCell(content) {
+  const cell = document.createElement('td');
+  cell.textContent = content;
+  return cell;
+}
+
+function calculateAge(person) {
+  return person.died - person.born;
+}
+
+function calculateCentury(year) {
+  return Math.ceil(year / 100 - 1);
+}
+
+function createRow(person) {
+  const row = document.createElement('tr');
+
+  const nameCell = createCell(person.name);
+  const sexCell = createCell(person.sex);
+  const bornCell = createCell(person.born);
+  const diedCell = createCell(person.died);
+  const ageCell = createCell(calculateAge(person));
+  const centuryCell = createCell(calculateCentury(person.died));
+
+  row.append(nameCell, sexCell, bornCell, diedCell, ageCell, centuryCell);
+
+  return row;
+}
```

Comment:
The `createRow` function is a good example of a function that does one thing and does it well. It's very clear what this function does: it creates a table row for a person. However, it could be made more reusable. Currently, it's tightly coupled to the structure of the `person` object. What if you wanted to create a row for a different kind of object? Consider making this function more generic, so it can create a row from any kind of object.